travis.sh

```diff
@@ -64,18 +64,19 @@ if [ ! "$CATKIN_PARALLEL_TEST_JOBS" ]; then export CATKIN_PARALLEL_TEST_JOBS="$C
 if [ ! "$ROS_REPOSITORY_PATH" ]; then export ROS_REPOSITORY_PATH="http://packages.ros.org/ros-shadow-fixed/ubuntu"; fi
 if [ ! "$ROSDEP_ADDITIONAL_OPTIONS" ]; then export ROSDEP_ADDITIONAL_OPTIONS="-n -q -r --ignore-src"; fi
 if [ ! "$CATKIN_TOOLS_BUILD_OPTIONS" ]; then
-  if [ "ROS_DISTRO" = "hydro" ]; then
+  if [ "$ROS_DISTRO" = "hydro" ]; then
     export CATKIN_TOOLS_BUILD_OPTIONS="-iv --summarize --limit-status-rate 0.002"
   else
     export CATKIN_TOOLS_BUILD_OPTIONS="--summarize --limit-status-rate 0.002"
   fi
 fi
+if [ "$SUDO_PIP" = false ]; then export SUDO_PIP=""; else export SUDO_PIP="sudo -H"; fi
 echo "Testing branch $TRAVIS_BRANCH of $REPOSITORY_NAME"
 # Setup pip
 # FIXME: need to specify pip version to 6.0.7 to avoid unexpected error
 # https://github.com/jsk-ros-pkg/jsk_robot/pull/523#issuecomment-164699366
-sudo easy_install 'pip==6.0.7'
-sudo pip install -U -q pip setuptools
+$SUDO_PIP easy_install 'pip==6.0.7'
+$SUDO_PIP pip install -U -q pip setuptools
```

Comment:
> I think this part is easy for developers to expect SUDO_PIP=sudo or empty.

But the `SUDO_PIP` variable cannot be reused, right?
Because it is overwritten from `"true"/"false"` to `"sudo"/""`

>  I think it could occur more difficult behavior e.g. SUDO_PIP is true and --as-root pip:yes is set to ROSDEP_ADDITIONAL_OPTIONS.

It seems that `--as-root` option is overwritten if there is duplicated options

```
% rosdep install --from-path . -r --as-root pip:no --simulate --reinstall --as-root pip:yes | grep pip
#[pip] Installation commands:
  sudo -H pip install -U slacker-cli
#[pip] Installation commands:
  sudo -H pip install -U chainer
#[pip] Installation commands:
  sudo -H pip install -U fcn
#[pip] Installation commands:
  sudo -H pip install -U percol
  sudo -H pip install -U freezegun
#[pip] Installation commands:
  sudo -H pip install -U gdown
#[pip] Installation commands:
  sudo -H pip install -U pygithub3
#[pip] Installation commands:
  sudo -H pip install -U dlib
#[pip] Installation commands:
  sudo -H pip install -U click
```